Header  header       # timestamp

string  power        # ON/OFF state
string  color        # the color value, RGB "255:255:255"

int64   brightness   # the brightness value, range from 1 to 100
